src/ssl.c

```diff
@@ -13258,6 +13258,7 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
                 #endif
                 if (ssl->options.sendVerify) {
                     if ( (ssl->error = SendCertificate(ssl)) != 0) {
+                        ProcessReplyEx(ssl, 0); /* See if an alert was sent. */
```

Comment:
Did you confirm if this also happens with TLS 1.3? It looks like there might be additional places in tls13.c that could need this.

Reply:
I didn't but should have. I'll check on that and make changes accordingly if needed.

Reply:
As I said in another comment. I think that ignoring the ProcessReplyEx return isn't a good idea and we should check for alerts on a socket error. For other errors (like a want_write) we want to return to the user asap.